cid/queries/ahq-queries.json

```diff
@@ -1,12 +0,0 @@
-{
```

Comment:
why these files removed? We still making this checks for customers who deploying dashboards over CUR tables not created by our CFN

Reply:
These are not used since a long time. These check are replaced by working with glue metadata objects.